astropy/convolution/tests/test_convolve_fft.py

```diff
@@ -975,20 +975,15 @@ def test_input_unmodified_with_nan(
     assert np.all(np.isnan(y[kernel_is_nan]))
 
 
-@pytest.mark.parametrize(
-    "error_kwarg", [{"psf_pad": True}, {"fft_pad": True}, {"dealias": True}]
-)
+@pytest.mark.parametrize("error_kwarg", ["psf_pad", "fft_pad", "dealias"])
 def test_convolve_fft_boundary_wrap_error(error_kwarg):
     x = np.array([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0], [7.0, 8.0, 9.0]], dtype=">f8")
     y = np.array([[1.0]], dtype=">f8")
-    assert (convolve_fft(x, y, boundary="wrap") == x).all()
-
-    with pytest.raises(ValueError) as err:
-        convolve_fft(x, y, boundary="wrap", **error_kwarg)
-    assert (
-        str(err.value)
-        == f"With boundary='wrap', {list(error_kwarg.keys())[0]} cannot be enabled."
-    )
+    assert np.all(convolve_fft(x, y, boundary="wrap") == x)
```

Comment:
If we are changing this line then we should change it to
```suggestion
    np.testing.assert_array_equal(convolve_fft(x, y, boundary="wrap"), x)
```
because it produces better output if the test should fail. For example, `assert np.all(convolve_fft(x, y, boundary="wrap") == x / 2)` produces the following:
```
E       assert False
E        +  where False = <function all at 0x7fcebeaa2170>(array([[1., 2...[7., 8., 9.]]) == array([[0.5, ...5, 4. , 4.5]])
E        +    where <function all at 0x7fcebeaa2170> = np.all
E           Use -v to get more diff)
```
but `np.testing.assert_array_equal(convolve_fft(x, y, boundary="wrap"), x / 2)` produces
```
E       AssertionError: 
E       Arrays are not equal
E       
E       Mismatched elements: 9 / 9 (100%)
E       Max absolute difference: 4.5
E       Max relative difference: 1.
E        x: array([[1., 2., 3.],
E              [4., 5., 6.],
E              [7., 8., 9.]])
E        y: array([[0.5, 1. , 1.5],
E              [2. , 2.5, 3. ],
E              [3.5, 4. , 4.5]])
```

Reply:
✅